### ISSUE TITLE
Everpub for clinical trial pubs?

### ISSUE THREAD

**Issue by aleimba** [everpub/openscienceprize#114]
First of great project idea, thanks to all involved.

I was thinking if Everpub could be a nice option for publications of clinical trials. Sadly, I'm not an expert myself, but there's a fair amount of statistical analyses in these publications. Also, the "significance" of these studies can be quite close to the "thresholds".

For an example of a typical publication, see here: https://www.ncbi.nlm.nih.gov/pubmed/26406150

AFAIK, clinical trials nowadays have to be registered beforehand (https://clinicaltrials.gov/), this includes predetermining the outcome to inhibit statistical mischief. There are initiatives, e.g. AllTrails (http://www.alltrials.net/) from Ben Goldacre, to report the results of clinical trials openly.

So, first off Everpub would be great to play around with the results of a Trial pub, e.g. trying different statistical tests/visualisations on the data etc.. And secondly it might be a tool of post-publication peer review if clinical trial data with the associated metadata can be pulled automatically.

**Comment by khinsen** [comment]
As far as I know, the only reproducibility issue specific to clinical trials is the confidentiality of patient data. Otherwise it's plain statistics, so Everpub ought to be a good fit if the data can be anonymized to permit publication.

**Comment by betatim** [comment]
I think something like everpub would be ideal for this. Do you know what kind of software people typically use for analysing drug trials data? We plan to make everpub play well with `R` which I think catches a lot of new users.

We should find someone who is an expert/active in this field to help with anonymisation and typical use cases if we want to target this explicitly.

**Comment by aleimba** [comment]
Yes, anonymization is important. But all the data should be anonymized already before storing in public repositories.

I think `R` would be ideal, it has all the tools needed. But, I don't know what trial data analysts use, probably an assortment of proprietary software and Excel ;-).

Also, stat analysis of clinical trials are quite structured (see e.g. FAQ 5 http://www.alltrials.net/find-out-more/faq/). A reference point should probably be the CONSORT (Consolidated Standards of Reporting Trials) 25-item checklist mentioned on the AllTrials website (http://www.consort-statement.org/checklists/view/32-consort/66-title). Thus, I can imagine working out a template with the typical items and stats (including some initial analysis for testing distribution, Q-Q plots, etc.). If the data and metadata is clean (which might be an issue), I guess a lot could be automated.

I don't know anybody directly involved in data analysis of trials. But maybe AllTrials/Ben Goldacre could be contacted directly? They should have (or know) people: http://www.alltrials.net/get-involved/

Thanks for picking this up guys!

**Comment by aleimba** [comment]
As a short sidenote: This meta-analysis pub might also serve as a reference point: http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0132382

They used a software called STATA (http://www.stata.com/) for statistics (see the additional methods section: http://journals.plos.org/plosone/article/asset?unique&id=info:doi/10.1371/journal.pone.0132382.s008).
